src/test/scala/com/fulcrumgenomics/umi/FilterConsensusReadsTest.scala

```diff
@@ -290,6 +290,67 @@ class FilterConsensusReadsTest extends UnitSpec {
     }
   }
 
+  private case class ReadNames(in: Seq[String], out: Seq[String])
+
+  private def sortOrderTest(name1: String, start1R1: Int, start1R2: Int, name2: String, start2R1: Int, start2R2: Int,
+                            inOrder: SamOrder, outOrder: Option[SamOrder] = None): ReadNames = {
```

Comment:
I think that in order to test more combinations it would be better to take `inSort` and `inGroup` separately as `SamOrder` has a limited set of combinations.